tests/performance_sitespeed_io.py

```diff
@@ -152,6 +153,9 @@ def validate_on_mobile_using_validator(url, validator_config):
     if 'nt' in os.name:
         arg = '--shm-size=1g -b chrome --mobile true --chrome.CPUThrottlingRate 3 --connectivity.profile 3gfast --visualMetrics true --plugins.remove screenshot --speedIndex true --browsertime.videoParams.createFilmstrip false --browsertime.chrome.args ignore-certificate-errors -n {0} --preScript chrome-custom.cjs {1}{2}'.format(
             get_config_or_default('SITESPEED_ITERATIONS'), url, browertime_plugin_options)
+    if 'docker' == WEBPERF_RUNNER:
+        arg = '--shm-size=1g -b chrome --mobile true --chrome.CPUThrottlingRate 3 --connectivity.profile 3gfast --visualMetrics false --plugins.remove screenshot --speedIndex false --browsertime.videoParams.createFilmstrip false --browsertime.chrome.args ignore-certificate-errors -n {0} --preScript chrome-custom.cjs {1}{2}'.format(
```

Comment:
I'm following https://github.com/sitespeedio/sitespeed.io/blob/main/Dockerfile-slim - I think in order to use both those a larger more complex image is needed that can run video. When I looked at the example in sitespeed.io repo that meant a full ubuntu-image and then it would be more troublesome to install the node and python versions needed.